db/migrate/20240111000939_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.string :text
+      t.bigint :UserId
+      t.bigint :PostId
```

Comment:
- [ ] Hello, you're doing great! 🌟 In alignment with Rails conventions, let's ensure we use snake_case naming for the table columns. Specifically, for consistency, modify the column names in your migration to adhere to this convention: